demo/src/routes/[framework]/getting-started/installation/fwk/svelte/NewProject.svelte

```diff
@@ -0,0 +1 @@
+<a href="https://kit.svelte.dev/docs/creating-a-project" target="_blank">SveltKit</a>
```

Comment:
```suggestion
<a href="https://svelte.dev/docs/introduction" target="_blank">Svelte</a>
```
Shouldn't you put a link to Svelte instead of SvelteKit?